src/pages/legal/terms.ejs

```diff
@@ -4,15 +4,74 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/1.1.0/marked.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/sanitize-html/1.26.0/sanitize-html.min.js"></script>
   <title>Terms - Universe List</title>
-  <%- include ("../parts/head.ejs") %>
+  <% include ("../parts/head.ejs") %>
 </head>
 <%- include ("../parts/navbar.ejs") %>
 <br><br><br><br>
 <div class="page-container">
   <div class="container">
     <div class="md" id="md">
-      <h2 class="heading"><b>Universe List Terms</h2><br><br>
-      <p>This is not a bug! Ishaan is just lazy<p>
+      <h2 class="heading"><b>Terms of Service</h2>
+      <br><br>
+      <div class="py-4">
+        <h3 class="heading-lg mb-2">What is ToS?</h3>
+        <p>
+          Terms of service (ToS) are a sort of document that specifies what a service provider is liable for as well as
+          user duties that must be met in order for the service to continue. Users who violate the terms of service are
+          subject to termination. Many websites and applications make their terms of service available.
+        </p>
+      </div>
+
+      <div class="py-4">
+        <h4 class="heading-lg">1. Using Our Website</h4>
+        <p>
+          A. Who can use UniverseList?
+        </p>
+        <p>
+          This website is for Discord Bot developers and users who want to grow their Discord servers. We conduct several updates on a daily and weekly basis to boost the security that you can have while also improving your experience with our website. You must provide us with accurate and complete information when creating your account.
+        </p>
+      </div>
+
+      <div class="py-4">
+        <h4 class="heading-lg">2. Your Content</h4>
+        <p>
+          A. Posting content
+        </p>
+        <p>
+          From the moment you post something on the website, we save your profile picture, user ID, username, and discriminator.
+        </p>
+        <p>
+          B. How long do we retain your data?
+        </p>
+        <p>
+          We keep your data when you have posted content on our website. When you don't have any content or delete them, our programmed system automatically deletes everything. This is not a cause for concern because the information obtained is obtained for security purposes as well as for a future scheduled backup.
+        </p>
+       <p>C. Location Information</p>
```

Comment:
Universe List doesnt store IP data or geolocation data

Reply:
Sorry about that,I'll fix it later